.github/workflows/ci.yml

```diff
@@ -19,62 +19,118 @@ jobs:
         include:
           - operating-system: 'ubuntu-20.04'
             php-version: '7.4'
-            job-description: 'with lowest deps'
+            job-description: 'tests with Symfony ^5'
+            run-tests: 'yes'
+            execute-flex-with-symfony-version: '^5' # explicit check for Symfony 5.x compatibility
+
+          - operating-system: 'ubuntu-20.04'
+            php-version: '7.4'
+            job-description: 'Fixer with lowest deps'
+            fixer-mode: 'standard'
             composer-flags: '--prefer-stable --prefer-lowest' # should be checked on the lowest supported PHP version
 
           - operating-system: 'ubuntu-20.04'
             php-version: '7.4'
-            job-description: 'with PHPDoc to type rules'
-            phpdoc-to-type-rules: 'yes' # should be checked on the lowest supported PHP version
+            job-description: 'Fixer with PHPDoc to type rules'
```

Comment:
original purpose was to check if Fixer is still operating properly after converting those phpdoc-types to native types

ref https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/actions/runs/6605602880/job/17940938286

<img width="525" alt="Screenshot 2023-10-25 at 00 34 00" src="https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/assets/2716794/ed79b965-28cf-46b3-a355-60e2dbd6704a">

Reply:
now, I think it only applies the types conversion, but does not execute anything further with it - ie not running the newly generated version of codebase

ref https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/actions/runs/6619129235/job/17979003746

<img width="525" alt="Screenshot 2023-10-25 at 00 35 09" src="https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/assets/2716794/83a5afef-4697-495b-8db1-6871162af010">

Reply:
The original idea may be good, but the implementation seems to be off? It was originally introduced in bd8ec6a3 and from the beginning it was only a `--dry-run`, so the consequent job is running against exactly the same codebase as `Symfony ^5` job. That's why I did it this way, because I thought it's only additional job and that's why Fixer is run 3 times for PHP 7.4 on my branch.

If I understand correctly, `Run PHP CS Fixer with PHPDoc to type rules` should _apply_ changes and then run regular Fixer configuration against modified codebase?

Reply:
Ha, thanks for checking before blindly applying - we run it on _LOWEST_ PHP we support which means we shall NOT see any changes in this job - if we detect that some types are not applied to codebase, we should fail the job (instead of applying it and running tests "what if types would be there")

Good discussion. We are getting somewhere.

Reply:
So it seems like we don't need to run Fixer _after_ this, in the same job, but can tun these 2 checks in separate job, right? I believe we don't gain much with merging these, while having it in the separate jobs gives us explicit feedback which one failed.

Reply:
yes for LOWEST, no for HIGHEST